app/controllers/tas_controller.rb

```diff
@@ -95,4 +95,17 @@ def upload_ta_list
   def user_params
     params.require(:user).permit(:user_name, :last_name, :first_name)
   end
+
+  before_action :set_assignment_ta, only: [:refresh_graph]
+
+  def set_assignment_ta
+    @assignment = Assignment.find(params[:assignment])
+    @current_ta = Ta.find(params[:id])
+  end
+
+  def refresh_graph
+    respond_to do |format|
```

Comment:
We generally haven't used `respond_to` with only one format; you can remove this.